addons/website_slides/static/src/js/public/components/course_tag_add_dialog/course_tag_add_dialog.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+  <t t-name="website_slides.CourseTagAddDialog">
+    <Dialog size="'md'" title="title">
+        <t t-set-slot="footer">
+            <a role="button" class="btn btn-primary me-1" t-on-click.prevent="onClickFormSubmit">
+                <span>Add</span>
+            </a>
+            <button type="button" class="btn btn-secondary" t-on-click="() => this.props.close()">
+                <span>Back</span>
+            </button>
+        </t>
+        <div class="mb-3">
+            <div t-if="state.alertMsg" t-out="state.alertMsg" class="alert alert-warning"/>
+            <div class="mb-3" t-att-class="{'form-control is-valid': validation.tagIsValid, 'form-control is-invalid': validation.tagIsValid === false}">
+                <label class="col-form-label">Tag</label>
+                <SelectMenu
+                    choices="choices.tagIds"
+                    onSelect.bind="onTagSelect"
+                    required="true" 
+                    togglerClass="'text-dark pe-1'"
+                    value="choices.tagId"
+                >
+                    <t t-out="displayTagValue"/>
+                    <t t-set-slot="bottomArea" t-slot-scope="select">
+                        <DropdownItem
+                            t-if="select.data.searchValue and state.canCreateTag"
+                            class="'btn text-primary'"
+                            onSelected="() => this.createChoice(select.data.searchValue)"
+                        >
+                            Create this tag "<i t-out="select.data.searchValue" />"
+                        </DropdownItem>
+                    </t>
+                </SelectMenu>
+            </div>
+            <div t-if="state.showTagGroup" t-att-class="{'mb-3': true, 'form-control is-valid': validation.tagGroupIsValid, 'form-control is-invalid': validation.tagGroupIsValid === false}">
```

Comment:
comment not solved here

![image](https://github.com/odoo/odoo/assets/54316226/7818dc58-2b9d-4087-bb11-0d293e76be43)

Reply:
My bad :pray: